package.json

```diff
@@ -23,38 +23,37 @@
     "lint:eslint": "eslint \"**/*.{ts,tsx}\" --max-warnings=0",
     "lint:tsc": "tsc",
     "lint:prettier": "prettier --cache --check .",
-    "analyze": "npx source-map-explorer 'build/static/js/*.js'"
+    "analyze": "pnpm dlx source-map-explorer \"dist/assets/*.js\" --no-border-checks"
   },
   "dependencies": {
-    "@h5web/app": "13.0.0",
-    "@h5web/h5wasm": "13.0.0",
-    "@react-hookz/web": "15.1.0",
+    "@h5web/app": "14.0.0",
+    "@h5web/h5wasm": "14.0.0",
+    "@react-hookz/web": "25.1.0",
     "h5wasm-plugins": "0.0.3",
-    "immer": "9.0.15",
     "normalize.css": "8.0.1",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "react-dropzone": "14.2.1",
-    "react-error-boundary": "4.0.11",
-    "react-hook-form": "7.34.2",
-    "react-icons": "4.4.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
+    "react-dropzone": "14.3.8",
+    "react-error-boundary": "5.0.0",
+    "react-hook-form": "7.54.2",
```

Comment:
10 pages of release notes, but our use of React Hook Form is quite basic, so again, nothing really relevant. For information, I did note one significant new feature, which is the ability to pass external, [reactive `values`](https://github.com/react-hook-form/react-hook-form/releases/tag/v7.41.0) to the form, which saves doing it via a `useEffect`.